.github/workflows/cml-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Generate CML Report
+on:
+  pull_request:
+    # types: [ labeled ]
+    branches:
+      - master
+
+jobs:
+  generate_cml_report:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container: 
+      image: ghcr.io/moja-global/rest_api_gcbm:master
+      ports:
+        - "8080:8080"
+    steps:
+    
+      - name: Checkout repository
+        uses: actions/checkout@v3
+          
+      - uses: actions/setup-node@v3
+        with:

```

Comment:
I think `rest_api_gcbm` already has node installed.

Reply:
It is but CML requires a higher version of Node which leads the action to fail in case we miss this step. I would go ahead and create a good first issue for this.